dev/insights/galaxy_ng.env

```diff
@@ -1,9 +1,7 @@
-PULP_CONTENT_PATH_PREFIX=/api/automation-hub/v3/artifacts/collections/
-
 PULP_GALAXY_API_PATH_PREFIX=/api/automation-hub/
 PULP_GALAXY_AUTHENTICATION_CLASSES=['galaxy_ng.app.auth.auth.RHIdentityAuthentication']
 PULP_GALAXY_PERMISSION_CLASSES=['rest_framework.permissions.IsAuthenticated', 'galaxy_ng.app.auth.auth.RHEntitlementRequired']
 PULP_RH_ENTITLEMENT_REQUIRED=insights
 
 PULP_ANSIBLE_API_HOSTNAME=http://localhost:5001
-PULP_ANSIBLE_CONTENT_HOSTNAME=http://localhost:24816/api/automation-hub/v3/artifacts/collections
```

Comment:
Same here. This parameter was requested by Towen team, because they were using pulp API directly.